docs/generator.md

```diff
@@ -39,6 +39,7 @@ minified               | boolean  | `false`         | Should the output be minif
 concise                | boolean  | `false`         | Set to `true` to reduce whitespace (but not as much as `opts.compact`)
 filename               | string   |                 | Used in warning messages
 jsonCompatibleStrings  | boolean  | `false`         | Set to true to run `jsesc` with "json": true to print "\u00A9" vs. "©";
+numbers                | string   | `decimal`       | Set `numbers: octal` in `jsescOption` to get the octal integer representation of any numeric value. Other supported options are `decimal, binary` and `hexadecimal`.
```

Comment:
I think we should document `jsescOption` instead as this table is for the top level generator options. We can link to https://github.com/mathiasbynens/jsesc#api for the shape of `jsescOption`.

Reply:
Made the changes. PTAL.